github-backup.py

```diff
@@ -8,15 +8,30 @@
 """
 
 
-from github import Github
-from argparse import ArgumentParser
+import os
+import errno
+import codecs
+import json
+import itertools
 import subprocess
-import os, os.path
 import logging
+import getpass
+try: #PY3
+    from configparser import SafeConfigParser as ConfigParser
+except ImportError:
+    from ConfigParser import SafeConfigParser as ConfigParser
+from argparse import ArgumentParser
+
+import requests
+import github
```

Comment:
I guess we need to import `pygithub3` here. See #32 as well as the example at https://pypi.org/project/pygithub3/

Reply:
pygithb3 is super old and unmaintained. We don't want to use it. In fact, it seems like you moved away from it in commit ac0f18d7b3 to the newer and maintained [PyGithub](https://pypi.org/project/PyGithub)